src/web/mod.rs

```diff
@@ -191,27 +191,45 @@ impl Handler for CratesfyiHandler {
     }
 }
 
+struct MatchVersion {
+    /// Represents the crate name that was found when attempting to load a crate release.
+    ///
+    /// `match_version` will attempt to match a provided crate name against similar crate names with
+    /// dashes (`-`) replaced with underscores (`_`) and vice versa. If
```

Comment:
Looks like this doc-comment got cut off.
```suggestion
    /// dashes (`-`) replaced with underscores (`_`) and vice versa.
```

Reply:
It was like that in #348 too :shrug:, guess I could fix it.